### ISSUE TITLE
Hidden elements should be removed from the DOM

### ISSUE THREAD

**Issue by OddDev** [Team-One-Developers/website#25]
Code setting the opacity instead of removing the element from the dom should be refactored to `display: none` etc. This is an a11y concern otherwise.
